name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/dataseries/DataSeriesSerializer.java

```diff
@@ -49,12 +49,16 @@ private void load(DataSeriesSet set, String config, List<DataSeries> series)
             {
                 series.add(s);
 
-                if (data.length == 4)
+                if (data.length >=4 && data.length <=5)
                 {
                     s.setColor(ColorConversion.hex2RGB(data[1]));
                     s.setLineStyle(LineStyle.valueOf(data[2]));
                     s.setShowArea(Boolean.parseBoolean(data[3]));
                 }
+                if(data.length == 5)
```

Comment:
Here I would suggest to change the condition to ``data.length >= 5``. This has the same advantage as described above:
Then it's quite easier to add more data items in the future without checking the old conditions. The condition can remain as long as data fields are added.